idsvr/README.md

```diff
@@ -136,6 +137,8 @@ paths:
 - /bar
 ```
 
+<b id="f3">3</b> The `unattendedinstall` script runs by default on the admin node if an admin `PASSWORD` is set, either by the value or some other environment variable. The installer creates default keys and enables the Admin UI so this configuration option shall be used if that config is either not necessary or loaded in some other way (i.e using `curity.config.configuration` or embedded in the docker image used).
```

Comment:
Did you mean "so this configuration option shall **NOT** be used" or am I reading this incorrectly :) ?

Reply:
no i think it is correct, maybe not clear enough. the configuration option (skipInstall) shall be used if the <generated> config is not necessery or loaded some other way